src/applications/claims-status/components/AddFilesForm.jsx

```diff
@@ -88,6 +89,11 @@ class AddFilesForm extends React.Component {
         errorMessage:
           'The file you selected is larger than the 25MB maximum file size and could not be added.',
       });
+    } else if (isEmptyFileSize(file)) {
+      this.setState({
+        errorMessage:
+          'The file you selected is empty. Files uploaded must be larger than 0B.',
```

Comment:
I think this should be spelled out: `0 bytes`

Reply:
Oops, I missed this comment. I'll address this in the next PR. The only reason I did it '0B' is because we do '25MB' for explaining the maximum. Maybe I should ask design.